matching-service/model/match-info-model.js

```diff
@@ -0,0 +1,25 @@
+import { DataTypes } from "sequelize";
+import { sequelize } from "./repository.js";
+
+/**
+ * Stores the information about matches made.
+ * Property: usernameOne < usernameTwo
+ */
+export const MatchInfo = sequelize.define("MatchInfo", {
```

Comment:
Since we are storing match info as logs, can we add a datetime column to additionally record the time at which this match is found? It would be helpful to debug and allow us to differentiate different sessions between the same users.

Reply:
I assumed it would create it by default, but I think it would be better to make it explicit. Will update, thank you!